src/components/common/ratio/Ratio.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect, useCallback } from "react";
+import "twin.macro";
+import "styled-components/macro";
+import * as ucum from "@lhncbc/ucum-lhc";
+import Quantity from "../quantity/Quantity";
+
+interface RatioProps {
+  label: string;
+  numeratorQuantity: number;
+  numeratorQuantityUnit: any;
+  denominatorQuantity: number;
+  denominatorQuantityUnit: any;
```

Comment:
can we not accept a single Ratio object instead of denom and numer separately?

Reply:
and I think we can do the same for the Quantity component?

Reply:
@adongare Quantity will be used for Quantity Interval as well, so it's a shared component.